emperor/support_files/js/controller.js

```diff
@@ -66,23 +94,39 @@ EmperorController.prototype.resize = function(width, height){
   // the area we have to present the plot is smaller than the total
   var plotWidth = this.$plotSpaceId.width();
 
+  // TODO: The below will need refactoring
+  // This is addressed in issue #414
```

Comment:
Ah I see what you are saying, ok to leave, but perhaps pointing at #405.
As that's where this should be fixed.

On (Aug-19-15|11:53), mortonjt wrote:
> > @@ -66,23 +94,39 @@ EmperorController.prototype.resize = function(width, height){
> >    // the area we have to present the plot is smaller than the total
> >    var plotWidth = this.$plotSpaceId.width();
> >  
> > +  // TODO: The below will need refactoring
> > +  // This is addressed in issue #414
> 
> If I remove this line, can I reopen the issue?  I think that refactoring is critical and I want to have a reminder somewhere
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ElDeveloper/emperor/pull/1/files#r37453006